conga-aem-definitions/src/main/roles/aem-dispatcher.yaml

```diff
@@ -133,6 +143,7 @@ files:
   variants:
   - aem-author*
   - ssl*
+  - http2*
```

Comment:
this is wrong - this would generate the whole config file only if all of aem-author, ssl and http2 variants are enabled. just remove this addition, should work without.